NuRadioReco/framework/trigger.py

```diff
@@ -341,3 +343,34 @@ def __init__(self, name, passband, order, threshold, number_of_coincidences=2,
         self._threshold = threshold
         self._number_of_coincidences = number_of_coincidences
         self._coinc_window = channel_coincidence_window
+
+class RNOGSurfaceTrigger(Trigger):
+    from NuRadioReco.utilities import units
+    def __init__(self, name, threshold, number_of_coincidences=1,
+                 channel_coincidence_window=60*units.ns, channels=[13, 16, 19], temp=250*units.kelvin, Vbias=2*units.volt):
+        """
+        initialize trigger class
+
+        Parameters
+        ----------
+        name: string
+            unique name of the trigger
+        threshold: float
+            the threshold
+        number_of_coincidences: int
+            the number of channels that need to fulfill the trigger condition
+            default: 1
+        channel_coincidence_window: float or None (default)
+            the coincidence time between triggers of different channels
+        channels: array of ints or None
+            the channels that are involved in the trigger
+            default: None, i.e. all channels
+        temp: temperature of the trigger board
+        Vbias: bias voltage on the trigger board
```

Comment:
docstring nitpicking - the parameters should be described
```
name: type
   description
```
as elsewhere.

Reply:
Nice one :) I change it.